test/common/upstream/load_balancer_fuzz_base.cc

```diff
@@ -0,0 +1,201 @@
+#include "test/common/upstream/load_balancer_fuzz_base.h"
+
+#include "test/common/upstream/utility.h"
+
+namespace Envoy {
+namespace Upstream {
+
+namespace {
+
+constexpr uint32_t MaxNumHostsPerPriorityLevel = 256;
+
+} // namespace
+
+void LoadBalancerFuzzBase::initializeASingleHostSet(
+    const test::common::upstream::SetupPriorityLevel& setup_priority_level,
+    const uint8_t priority_level) {
+  const uint32_t num_hosts_in_priority_level = setup_priority_level.num_hosts_in_priority_level();
+  ENVOY_LOG_MISC(trace, "Will attempt to initialize host set {} with {} hosts.", priority_level,
+                 num_hosts_in_priority_level);
+  MockHostSet& host_set = *priority_set_.getMockHostSet(priority_level);
+  uint32_t hosts_made = 0;
+  // Cap each host set at 256 hosts for efficiency
+  const uint32_t max_num_hosts_in_priority_level = MaxNumHostsPerPriorityLevel;
+  // Leave port clause in for future changes
+  while (hosts_made < std::min(num_hosts_in_priority_level, max_num_hosts_in_priority_level) &&
+         port_ < 65535) {
+    host_set.hosts_.push_back(makeTestHost(info_, "tcp://127.0.0.1:" + std::to_string(port_)));
+    ++port_;
+    ++hosts_made;
```

Comment:
what was this used for?

Reply:
The for loop iteration?

Reply:
oh, right, ignore my comment -- it's just not used anywhere outside the while* loop.
Maybe it should be a for loop, but up to you

Reply:
While loop*